patches/ungoogled-chromium/build.patch

```diff
@@ -0,0 +1,11 @@
+--- a/build/config/compiler/BUILD.gn
++++ b/build/config/compiler/BUILD.gn
+@@ -652,7 +652,7 @@
+       # and let it use all cores.
+       # TODO(thakis): Check if '=0' (that is, number of cores, instead
+       # of "all" which means number of hardware threads) is faster.
+-      ldflags += [ "-Wl,--thinlto-jobs=all" ]
++      ldflags += [ "-Wl,--thinlto-jobs=8" ]
```

Comment:
Do we need this patch if we use LLVM 11?

Reply:
Apparently not.
I was planning to make two PR, but one probably make more sense, you are right.